src/test/pmem_map_file/TEST10.PS1

```diff
@@ -0,0 +1,54 @@
+#
+# Copyright 2014-2019, Intel Corporation
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+#     * Redistributions of source code must retain the above copyright
+#       notice, this list of conditions and the following disclaimer.
+#
+#     * Redistributions in binary form must reproduce the above copyright
+#       notice, this list of conditions and the following disclaimer in
+#       the documentation and/or other materials provided with the
+#       distribution.
+#
+#     * Neither the name of the copyright holder nor the names of its
+#       contributors may be used to endorse or promote products derived
+#       from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+# LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+# A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+# OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+# SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+# LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+# DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+# THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+
+#
+# src/test/pmem_map_file/TEST10 -- unit test for pmem_map_file
+#
+
+# standard unit test setup
+. ..\unittest\unittest.ps1
+
+require_test_type medium
+require_fs_type any
+
+setup
+
+# <path> <size> <flags> <mode> <use_mlen> <use_is_pmem> <err_code>
+
+expect_normal_exit $Env:EXE_DIR\pmem_map_file$Env:EXESUFFIX `
+	$DIR\testfile1 4096 CES 0666 0xdeadbeef 0xdeadbeef 0
```

Comment:
0xdeadbeef ?

Use_is_pmem is true/false argument

Reply:
“deadbeef” (in nvml_val_tools) was used to initialize the parameter with garbage information to be able to check if the value was changed.
I can see now that the test structure is different in PMDK so that doesn't make sense anymore. 
Changed.